doc/manuals/user/transient_entities.md

```diff
@@ -0,0 +1,124 @@
+# Transient Entities
+
+Orion Context Broker implements *transient entities*. A transient entity is like any other entity 
+(i.e. it has id/type, a set of attributes, etc.) but with a limited lifetime, so when a given
+time has come, the entity is automatically deleted from the context managed by Orion.
+
+Thus, a first and very important piece of advice: **be careful if you use transient entities as once
+the expiration time has come, the entity will be automatically deleted from database and there is
+no way of recovering it**. Ensure the information you set in a transient entity is actually "transient"
+(i.e. it is not important after the lifetime of the entity has passed and it gets expired).
+
+In addition, **have a look to the [backward compatibility considerations section](#backward-compatibility-considerations)
+in the case you are already using attributes with the exact name `dateExpires`**.
+
+## The `dateExpired` attribute
+
+Transient entities are based in the `dateExpires` attribute. This is an attribute of `DateTime` type,
+according to the [NGSIv2 specification](http://telefonicaid.github.io/fiware-orion/api/v2/stable/). 
```

Comment:
Should transient entities functionality being included in NGSIv2? Or, in other words, should be this specification in the NGSIv2 .apib or in a Orion specific .md?

I think is better to keep it as an specific implementation aspect (i.e. in a Orion specific .md) in order to keep NGSIv2 spec simpler. However, I'm open to suggestions. @jmcanterafonseca what do you think?

Reply:
On a second thought, maybe `dataExpires` can be considered part of the "operational information" of the entity and, in that sense, should be a NGSIv2 system/builtin attribute, as `dateModified` or `dateCreated` (the original ideal... later discarded... now again under discussion :)